asset/css/page-nav.css

```diff
@@ -27,13 +25,22 @@
     color: black;
 }
 
-#page-nav-content-wrapper {
-    margin-right: 300px;
-    transition: 0.4s;
-    -webkit-transition: 0.4s;
+.page-nav-title {
+    color: black;
+    white-space: inherit;
+}
+
+.nested {
+    margin-left: 5%;
+}
+
+.no-flex-wrap {
+    flex-wrap: nowrap;
 }
 
-/* Responsive site navigation */
+
```

Comment:
Is the number of blank lines here intended?

Reply:
Not intended, looks like `eslint` did not complain about this :P

Reply:
ESLint doesn't check CSS files. :see_no_evil:

Reply:
Is there any interest in adding a linter for css? (e.g. https://stylelint.io) 🙂

Reply:
> Is there any interest in adding a linter for css? (e.g. https://stylelint.io) 🙂

@marvinchin PR is welcome for a css linter.